content/en/serverless/distributed_tracing/_index.md

```diff
@@ -2,48 +2,123 @@
 title: Distributed Tracing
 kind: documentation
 further_reading:
-    - link: 'tracing/serverless_functions'
-      tag: 'Documentation'
-      text: 'Installing AWS X-Ray'
+- link: "/tracing/"
+  tag: "Documentation"
+  text: "Explore Datadog APM"
+- link: "/tracing/trace_search_and_analytics/#live-search-for-15-minutes"
+  tag: "Documentation"
+  text: "Live Search"
 ---
 
 {{< img src="tracing/serverless_functions/ServerlessDistributedTrace.png" alt="Trace Serverless Functions"  style="width:100%;">}}
 
-By connecting your serverless traces to metrics, Datadog provides a context-rich picture of your application’s performance, allowing you to better troubleshoot performance issues given the distributed nature of serverless applications.
+By connecting your serverless traces to metrics, Datadog provides a context-rich picture of your application’s performance, allowing you to better troubleshoot performance issues given the distributed nature of serverless applications. 
+
+The Datadog Python, Node.js, Ruby, Go and Java tracing libraries support distributed tracing for AWS Lambda. The easiest way to add tracing to your application is with the [Datadog Lambda Library][2], which includes the Datadog tracing library as a dependency.
 
 ## Choose your Tracing Library
 
 {{< img src="integrations/amazon_lambda/lambda_tracing.png" alt="architecture diagram for tracing AWS Lambda with Datadog" >}}
 
-Depending on your language and configuration, choose between setting up Datadog APM or AWS X-Ray for your tracing needs. [Read this page][1] to learn more about tracing with AWS X-Ray.
+To start using Datadog APM with your serverless application, you need to first choose between installing Datadog's tracing libraries (`dd-trace`) or AWS X-Ray tracing libraries based on your Lambda runtime and individual requirements. To see all of your traces in Datadog in real-time in the Live Search view, you need to use Datadog's tracing libraries.
 
-## Distributed Tracing with Datadog APM
+| [Datadog APM with dd-trace][1]          | [Datadog APM with AWS X-Ray][2]           |
+|---------------------------------|-------------------------------------------------------------------------|
+| Uses Datadog APM's integration libraries for end-to-end tracing.  | Pulls traces from AWS X-Ray. |
+| Visualize your traces in Datadog in real-time. | Trace data available in Datadog after 2-5 minutes. |
+| Tail-based sampling and fully customizable tag-based retention filters. | Sampling can be modified from the AWS X-Ray console. |
```

Comment:
`Tail-based sampling and fully customizable tag-based retention filters.` => `Fully customizable tag-based retention rules.`? I think sampling and retention mean the same thing.

Reply:
They are slightly different to my understanding:

Sampling is what picks which traces are most important to index (tail-based sampling is desirable because it means that traces with errors/high latency, etc.) will always be indexed.

Retention filters is how long your traces are indexed for.